geopmpy/geopmpy/launcher.py

```diff
@@ -106,6 +106,53 @@ def range_str(values):
     return ','.join(result)
 
 
+def get_cpuset():
+    """
+    Returns a list of valid CPUs for the current process.
+    Ported from C++ Helper::get_cpuset().
```

Comment:
Would be nice to have google doc style "Returns: " with the type

Reply:
Nevermind.  This is not done elsewhere in this file.  Better to stay with local style i guess